src/routes/docs/pages/quickstart.md

```diff
@@ -83,7 +83,7 @@ const config = {
     extend: {
       colors: {
         // flowbite-svelte
-         primary: {50:'#ebf5ff',100: '#fff1ee',200:'#ffe4de',300:'#ffd5cc',400:'#ffbcad',500:'#fe795d',600:'#ef562f',700:'#eb4f27',800:'#d3330a',900:'#d3330a'}
+         primary: {50:'#fff1ee',100: '#fff1ee',200:'#ffe4de',300:'#ffd5cc',400:'#ffbcad',500:'#fe795d',600:'#ef562f',700:'#eb4f27',800:'#d3330a',900:'#d3330a'}
```

Comment:
There's no difference between primary-50 and primary-100. As well primary-800 is the same as primary-900. Is that intentional?

Reply:
You're right, it seems that in Figma it's the same color. @robert1508 can you check this please?

<img width="1012" alt="Screenshot 2023-06-04 at 19 02 46" src="https://github.com/themesberg/flowbite-svelte/assets/8052108/e3d71a30-faec-4bdd-a6c3-6aed7d29004a">

Reply:
Everything is revised and I updated these colors, I added the color shades for primary-50, 800, and 900, and now everything is good to go.